client/src/components/Ticketing/ticketingmanager/showings/InstancesPage.tsx

```diff
@@ -46,13 +47,20 @@ const InstancesPage = () => {
         <ul className='md:grid md:grid-cols-2 md:gap-8 sm:grid sm:grid-cols-1 sm:gap-4 mt-9'>
           {allEvents.map((eventss) => (
```

Comment:
I know this wasn't you, but do you think you could switch over the use of `eventss` to `event` if it doesn't conflict with anything.

Reply:
Updated